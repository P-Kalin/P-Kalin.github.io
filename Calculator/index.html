<!doctype html>
<html lang="ru">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Calculator</title>
    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
    <link rel="stylesheet" href="./css/style.css">
</head>
<body>
    <div class="wrapper">
        <div id="calc" class="calc">
            <div class="calc-display">
                <div class="calc-display_field" id="filed"></div>
                <div class="calc-display_result" id="result"></div>
            </div>
            <div class="calc-keyboard">
                <div class="calc-keyboard_main">
                    <button class="calc-keyboard_main--key">7</button>
                    <button class="calc-keyboard_main--key">8</button>
                    <button class="calc-keyboard_main--key">9</button>

                    <button class="calc-keyboard_main--key">4</button>
                    <button class="calc-keyboard_main--key">5</button>
                    <button class="calc-keyboard_main--key">6</button>

                    <button class="calc-keyboard_main--key">1</button>
                    <button class="calc-keyboard_main--key">2</button>
                    <button class="calc-keyboard_main--key">3</button>

                    <button class="calc-keyboard_main--key">,</button>
                    <button class="calc-keyboard_main--key">0</button>
                    <button class="calc-keyboard_main--key">=</button>
                </div>
                <div class="calc-keyboard_aside">
                    <button class="calc-keyboard_aside--key">C</button>
                    <button class="calc-keyboard_aside--key">&#215;</button>
                    <button class="calc-keyboard_aside--key">&#247;</button>
                    <button class="calc-keyboard_aside--key">-</button>
                    <button class="calc-keyboard_aside--key">+</button>
                </div>
            </div>
        </div>
    </div>
    <script src="./js/main.js"></script>
</body>
</html>