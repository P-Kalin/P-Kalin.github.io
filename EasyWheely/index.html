<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<title>Easy Wheely</title>

	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/
	bootstrap/3.3.7/css/bootstrap.min.css" 
	integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" 
	crossorigin="anonymous">

	<link rel="stylesheet" href="css/main.css">
</head>
<body>
	<header>
		<div class="first">
			<div class="container">
				<div class="row">
					<div class="col-md-4">
						<img src="img/ew_logo.png">
					</div>
					<div class="col-md-6">
						<h1>EasyWheely <span>- удобная и быстрая<br> технология парковки</span></h1>
					</div>
					<div class="col-md-2">
						
					</div>
				</div>
			</div>
		</div>
		<div class="heading">
			<div class="container">
				<div class="row">
					<div class="col-md-12">
						<nav class="menu">
							<ul>
								<li><a href="parking.html">Припарковаться</a></li>
								<li><a href="#support">Контакты</a></li>
								<li><a href="index.html">Сотрудничество</a></li>
							</ul>
						</nav>
					</div>
				</div>
			</div>
		</div>
	</header>
	<section id="main__info">
		<div class="container">
			<div class="row">
				<div class="col-md-4">
					<div class="main__item">
						<p>Быстрая парковка</p>
						<img src="img/map.png">
					</div>
				</div>
				<div class="col-md-4">
					<div class="main__item">
						<p>Экономия времени</p>
						<img src="img/watch.png">
					</div>
				</div>
				<div class="col-md-4">
					<div class="main__item">
						<p>Удобное использование<br> в любом месте</p>
						<img src="img/car.png">
					</div>
				</div>
			</div>
		</div>
	</section>
	<section id="support">
		<div class="container">
			<img src="img/EW.png" alt="" class="ew_bg">
			<div class="row">
				<div class="support">
					<div class="col-md-offset-8 col-md-4">
						<img src="img/park_logo.png" class="img-responsive center-block">
						<p class="support__tech">Круглосуточная <br>техподдержка</p>
						<img src="img/phone_logo.png" class="img-responsive center-block support__numb_img">
						<p class="support__numb">
							+380 (97) 867 56 34<br>
							+380 (99) 679 56 66<br>
							+380 (67) 123 54 67<br>
						</p>
					</div>
				</div>			
			</div>
		</div>
	</section>
	<footer>
		<div class="container">
			<div class="row">
				<div class="col-md-12">
					<span>Copyright  © 2018 | Все права  защищены</span>	
				</div>
			</div>
		</div>
	</footer>

<script type="text/javascript" src="js/main.js"></script>	
</body>
</html>