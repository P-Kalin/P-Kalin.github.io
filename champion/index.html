<!DOCTYPE html>
<html lang="ru">
<head>
	<meta charset="UTF-8">
	<title>Чемпион</title>
	<meta name="viewport" content="initial-scale=1, maximum-scale=1">
	<link rel="stylesheet" href="css/bootstrap.min.css">
	<link rel="stylesheet" href="css/style.css">
	<link rel="stylesheet" href="css/fonts.css">
	<link rel="stylesheet" href="css/media.css">
</head>
<body>
	<header id="header">
		<nav class="navbar navbar-expand-lg fixed-top">
			<a class="navbar-brand" href="index.html"><img class="logo" src="img/logo.png" alt=""></a>
			<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar1" aria-controls="navbar1" aria-expanded="false" aria-label="Toggle navigation">
				<span class="navbar-toggler-icon"></span>
			</button>
			<div class="collapse navbar-collapse" id="navbar1">
				<ul class="navbar-nav mr-auto">
					<li class="nav-item active">
						<a class="nav-link" href="#header">Главная</a>
					</li>
					<li class="nav-item">
						<a class="nav-link" href="#about">О нас</a>
					</li>
					<li class="nav-item">
						<a class="nav-link" href="#contacts">Контакты</a>
					</li>
				</ul>
			</div>
		</nav>	
		<div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
			<ol class="carousel-indicators">
				<li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
				<li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
				<li data-target="#carouselExampleIndicators" data-slide-to="2"></li>
			</ol>
			<div class="carousel-inner text-center">
				<div class="carousel-item active carousel-item__first">
					<h1>
						Магазин автозапчастей Чемпион. <br>
						Мы не продаем "все" -<br>мы предлагаем лучшее
					</h1>
					<a href="#about"><button type="button" class="btn btn-secondary btn-lg">Подробнее</button></a>
				</div>
				<div class="carousel-item carousel-item__second">
					<h1>
						Широкий ассортитмент - <br>в наличие на складе более 10 000 наименований<br> запасных частей, масел и расходных материалов
					</h1>
					<a href="#about"><button type="button" class="btn btn-light btn-lg">Подробнее</button></a>
				</div>
				<div class="carousel-item carousel-item__third">
					<h1>Работаем с 2000 года. <br>Только проверенные производители, <br>ответственность за правильность подбора запчастей</h1>
					<a href="#about"><button type="button" class="btn btn-secondary btn-lg">Подробнее</button></a>
				</div>
			</div>
			<a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
				<span class="carousel-control-prev-icon" aria-hidden="true"></span>
				<span class="sr-only">Previous</span>
			</a>
			<a class="carousel-control-next" href="#carouselExampleIndicators" role="button" data-slide="next">
				<span class="carousel-control-next-icon" aria-hidden="true"></span>
				<span class="sr-only">Next</span>
			</a>
		</div>
	</header>
	<section id="details">
		<div class="details-items ">
			<ul class="d-flex flex-wrap">
				<li class="details-item">
					<div class="item-wrapper view overlay zoom">
						<span class="category-image"><img src="img/details_1.jpg" alt=""></span>
						<div class="item-content">
							<span class="item-content__title">Моторная группа</span>
							<span>Nüral, Glyco, MAHLE</span>
						</div>
					</div>
				</li>
				<li class="details-item">
					<div class="item-wrapper">
						<span class="category-image"><img src="img/details_2.jpg" alt=""></span>
						<div class="item-content">
							<span class="item-content__title">Амортизаторы</span>
							<span>SACHS, KYB, MONROE</span>
						</div>
					</div>
				</li>
				<li class="details-item">
					<div class="item-wrapper">
						<span class="category-image"><img src="img/details_3.jpg" alt=""></span>
						<div class="item-content">
							<span class="item-content__title">Тормозные системы</span>
							<span>TRW, FERODO, TEXTAR, REMSA, SANSING, BREMBO</span>
						</div>
					</div>
				</li>
				<li class="details-item">
					<div class="item-wrapper">
						<span class="category-image"><img src="img/details_4.jpg" alt=""></span>
						<div class="item-content">
							<span class="item-content__title">Выхлопная система</span>
							<span>BOSAL, Polmostrow</span>
						</div>
					</div>
				</li>
				<li class="details-item">
					<div class="item-wrapper">
						<span class="category-image"><img src="img/details_5.jpg" alt=""></span>
						<div class="item-content">
							<span class="item-content__title">Подвеска</span>
							<span>LEMFÖRDER, TRW, MOOG, CTR, SASIC</span>
						</div>
					</div>
				</li>
				<li class="details-item">
					<div class="item-wrapper">
						<span class="category-image"><img src="img/details_6.jpg" alt=""></span>
						<div class="item-content">
							<span class="item-content__title">Элементы трансмиссии</span>
							<span>SACHS, ZF, LuK</span>
						</div>
					</div>
				</li>
				<li class="details-item">
					<div class="item-wrapper">
						<span class="category-image"><img src="img/details_7.jpg" alt=""></span>
						<div class="item-content">
							<span class="item-content__title">Фильтра</span>
							<span>MANN, MAHLE, WIX, PURFLUX, CHAMPION</span>
						</div>
					</div>
				</li>
				<li class="details-item">
					<div class="item-wrapper">
						<span class="category-image"><img src="img/details_8.jpg" alt=""></span>
						<div class="item-content">
							<span class="item-content__title">Масла</span>
							<span>MOTUL, KROON OIL, MOBIL, TOTAL, ELF, CASTROL</span>
						</div>
					</div>
				</li>
			</ul>
		</div>
	</section>
	<section id="brends">
		<div class="container-fluid">
			<div class="row justify-content-around flex-nowrap text-center">
				<div class="brend-item"><a href=""><img src="img/brend-motul.png" alt=""></a></div>
				<div class="brend-item"><a href=""><img src="img/brend-brembo.png" alt=""></a></div>
				<div class="brend-item"><a href=""><img src="img/brend-bosch.png" alt=""></a></div>
				<div class="brend-item"><a href=""><img src="img/brend-garrett.png" alt=""></a></div>
			</div>
			<div class="row justify-content-around flex-nowrap text-center">
				<div class="brend-item"><a href=""><img src="img/brend-champion.png" alt=""></a></div>
				<div class="brend-item"><a href=""><img src="img/brend-monroe.png" alt=""></a></div>
				<div class="brend-item align-middle"><a href=""><img src="img/brend-castrol.png" alt=""></a></div>
				<div class="brend-item"><a href=""><img src="img/brend-lemforder.png" alt=""></a></div>
			</div>
		</div>
	</section>
	<!-- about us -->
	<section id="about">
		<div class="container-fluid">
			<div class="row">
				<div class="about col-lg-12 text-center">
					<h2>Почему именно мы?</h2>
					<div class="line"></div>
					<p>
						Большой опыт в подборе и рекомендациях для различных типов автомобилей,<br>
						только проверенные бренды. Ответственность за правильность подбора запчастей. <br>
						Гарантийные обязательства и разумный компромисс с покупателем
					</p>
				</div>
			</div>
		</div>
	</section>
	<!-- Contacts -->
	<section id="contacts">
		<div class="container-fluid">
			<div class="row text-center">
				<h2 class="contacts-title col-lg-12">Контакты</h2>

			</div>
			<div class="line"></div>
			<div class="row text-center justify-content-center">
				<div class="contacts-item col-lg-4 col-md-4">
					<img src="img/worker.png" alt="">
					<p>Время работы</p>
					<ul>
						<li>Пн-Пт: 9:00 - 18:00</li>
						<li>Сб: 9:00 - 17:00</li>
						<li>Вс: Выходной</li>
					</ul>
				</div>
				<div class="contacts-item col-lg-4 col-md-4">
					<img src="img/conversation.png" alt="">
					<p>Обратная связь</p>
					<ul class="contacts-item__phones">
						<li><a href="tel:+380977061494" class="phone">(097) 706-14-94</a></li>
						<li><a href="tel:061942-43-44" class="phone">(0619) 42-43-44</a></li>
						<li><a title="Viber" href="viber://chat?number=+380977061494" class="viber">(097) 706-14-94</a></li>
					</ul>
				</div>
				<div class="contacts-item col-lg-4 col-md-4">
					<img src="img/parked-car.png" alt="">
					<p>Адрес</p>
					<ul class="contacts-item__map">
						<li><a href="https://www.google.com/maps/place/%D0%A7%D0%B5%D0%BC%D0%BF%D0%B8%D0%BE%D0%BD/@46.8501345,35.36177,19.54z/data=!4m13!1m7!3m6!1s0x40c2b1e38fdbc575:0x8edb3a17b531262a!2z0YPQu9C40YbQsCDQmNC90YLQtdGA0LrRg9C70YzRgtGD0YDQvdCw0Y8sIDE0NSwg0JzQtdC70LjRgtC-0L_QvtC70YwsINCX0LDQv9C-0YDQvtC20YHQutCw0Y8g0L7QsdC70LDRgdGC0YwsIDcyMzAw!3b1!8m2!3d46.8499448!4d35.3626794!3m4!1s0x0:0xef86559c2dd51a3c!8m2!3d46.8502315!4d35.3619035" target="_blank">
							Украина, Запорожская обл., <br>г. Мелитополь, <br>ул. Интеркультурная 145
						</a></li>
					</ul>
				</div>
			</div>
		</div>
	</section>
	<footer>
		<div class="container-fluid">
			<p class="footer text-center">© 2019 Copyright. All Rights Reserved. Design by Kalin Pavel</p>	
		</div>
	</footer>
	
	<script
	src="https://code.jquery.com/jquery-3.3.1.min.js"
	integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
	crossorigin="anonymous"></script>
	<script src="js/script.js"></script>
	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
	<script src="js/bootstrap.min.js"></script>
</body>
</html>