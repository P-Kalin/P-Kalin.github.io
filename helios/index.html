<!DOCTYPE html>
<html lang="ru">
<head>
	<meta charset="UTF-8">
	<title>Автомастерская Хелиос</title>
	
	<!-- Latest compiled and minified CSS -->
	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/
	bootstrap/3.3.7/css/bootstrap.min.css" 
	integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" 
	crossorigin="anonymous">

	<link href="https://fonts.googleapis.com/css?family=
	htmlOpen+Sans:400,800&amp;subset=cyrillic" rel="stylesheet"> 

	<link rel="stylesheet" href="css/main.css">
	<link rel="stylesheet" href="css/media.css">
</head>
<body>
	<header>
		<div class="heading">
			<div class="container">
				<div class="row">
					<div class="col-md-3 col-lg-3 col-sm-6 col-xs-4">
						<img src="img/logo.png" alt="Хелиос" class="logo img-responsive"></img>
					</div>
					<div class="col-md-5 col-lg-6">
						<nav class="visible-lg">
							<ul class="menu clearfix">
								<li class="menu__item">
									<a href="#">
										Преимущества
									</a>
								</li>
								<li class="menu__item">
									<a href="#">
										Услуги
									</a>
								</li>
								<li class="menu__item">
									<a href="#">
										Наш центр
									</a>
								</li>
								<li class="menu__item">
									<a href="#">
										Отзывы
									</a>
								</li>
								<li class="menu__item">
									<a href="#">
										Контакты
									</a>
								</li>
							</ul>
						</nav>
					</div>
					<div class="col-md-4 col-lg-3 col-sm-6 col-xs-8 text_right">
						<div class="contacts">
							<div class="contacts__phone">
								+7 (495) 748-23-76
							</div>
							<a class="contacts__btn upper" href="#">Заказать звонок</a>
						</div>
					</div>	
				</div>
			</div>
		</div>
		<div class="first">
			<div class="container">
				<div class="row">
					<div class="col-md-8">
						<h1>Комплексная<br>
							автомастерская<br> 
							в Самаре
						</h1>
						<ul class="services clearfix">
							<li class="services__item services__item_fix pull-left">
								<div class="services__title">
									Ремонтный цех
								</div>
								<div class="services__text">
									Lorem ipsum dolor sit amet,
									cosectetur adipisicing elit,
									sed do eiusmod tempor incididunt
									cabo
								</div>
							</li>
							<li class="services__item services__item_frame pull-right">
								<div class="services__title">
									Кузовной цех
								</div>
								<div class="services__text">
									Lorem ipsum dolor sit amet,
									cosectetur adipisicing elit,
									sed do eiusmod tempor incididunt
									cabo
								</div>
							</li>
							<li class="services__item services__item_electric pull-left">
								<div class="services__title">
									Электрика
								</div>
								<div class="services__text">
									Lorem ipsum dolor sit amet,
									cosectetur adipisicing elit,
									sed do eiusmod tempor incididunt
									cabo
								</div>
							</li>
							<li class="services__item services__item_wash pull-right">
								<div class="services__title">
									Автомойка
								</div>
								<div class="services__text">
									Lorem ipsum dolor sit amet,
									cosectetur adipisicing elit,
									sed do eiusmod tempor incididunt
									cabo
								</div>
							</li>
						</ul>
					</div>
					<div class="col-md-4 col-sm-12">
						<div class="form">
							<div class="form__title upper text-center">
								Запишись на бесплатный осмотр
								Вашего авто
							</div>
							<form class="order" action="#">
								<div class="order__group">
									<label for="name">
										Ваше имя
									</label>
									<input type="text" name="name" class="order__input" 
									placeholder="Например, Михаил" required id="name">
								</div>
								<div class="order__group">
									<label for="tel">
										Ваш телефон
									</label>
									<input type="tel" name="tel" class="order__input" 
									placeholder="+7 (__) __-__-__" required id="tel">
								</div>
								<div class="order__group">
									<label for="auto">
										Марка / модель авто
									</label>
									<input type="text" name="auto" class="order__input" 
									placeholder="Например, Ford focus" required id="auto">
								</div>
								<button class="order_btn act" type="submit">Записаться</button>
								<div class="order__credits">
									Ваши данные под защитой
								</div>
							</form>
						</div>
					</div>
				</div>
			</div>
		</div>
	</header>
	<section id="why">
		<div class="container">
			<img src="img/car_bg.png" alt=" " class="car__bg">
			<div class="row">
				<div class="col-md-12">
					<h2>
						Почему нас выбирают
					</h2>
				</div>
			</div>
			<ul class="adv">
				<div class="row">
					<div class="col-md-3">
						<li class="adv__item text-center">
							<div class="adv__img adv__img_first"></div>
							<div class="adv__title upper">
								Контроль качества
							</div>
							<p class="adv__text">
								Lorem ipsum dolor sit amet,<br>
								cosectetur adipisicing elit,<br>
								sed do eiusmod tempor incididunt cabo<br>
								lorem ipsum dolor sit amet,<br>
								cosectetur adipisicing elit,<br>
								sed do eiusmod tempor incididunt cabo
							</p>	
						</li>
					</div>
					<div class="col-md-3">
						<li class="adv__item text-center">
							<div class="adv__img adv__img_second"></div>
							<div class="adv__title upper">
								Комплекс услуг
							</div>
							<p class="adv__text">
								Lorem ipsum dolor sit amet,<br>
								cosectetur adipisicing elit,<br>
								sed do eiusmod tempor incididunt cabo<br>
								lorem ipsum dolor sit amet,<br>
								cosectetur adipisicing elit,<br>
								sed do eiusmod tempor incididunt cabo
							</p>	
						</li>
					</div>
					<div class="col-md-3">
						<li class="adv__item text-center">
							<div class="adv__img adv__img_third"></div>
							<div class="adv__title upper">
								Оригинальные запчасти
							</div>
							<p class="adv__text">
								Lorem ipsum dolor sit amet,<br>
								cosectetur adipisicing elit,<br>
								sed do eiusmod tempor incididunt cabo<br>
								lorem ipsum dolor sit amet,<br>
								cosectetur adipisicing elit,<br>
								sed do eiusmod tempor incididunt cabo
							</p>	
						</li>
					</div>
					<div class="col-md-3">
						<li class="adv__item text-center">
							<div class="adv__img adv__img_fourth"></div>
							<div class="adv__title upper">
								Комфорт для клиентов
							</div>
							<p class="adv__text">
								Lorem ipsum dolor sit amet,<br>
								cosectetur adipisicing elit,<br>
								sed do eiusmod tempor incididunt cabo<br>
								lorem ipsum dolor sit amet,<br>
								cosectetur adipisicing elit,<br>
								sed do eiusmod tempor incididunt cabo
							</p>	
						</li>
					</div>
				</div>
			</ul>
		</div>
	</section>
	<section id="services">
		<div class="container">
			<div class="row">
				<div class="col-md-12">
					<h2>Какие услуги мы оказываем</h2>
				</div>
			</div>
			<div class="row">
				<div class="col-md-8">
					<div class="description">
						<p>
							В нашей мастерской вы получаете
							комплекс всех необходимых услуг - от
							технического обсуживания до
							модернизации вашего автомобиля. Наши
							мастера решат любые проблемы вашего
							авто, будь то диагностика и ремонт
							узлов-генераторов, установка и ремонт
							элеткрооборудования, или кузовной ремонт
						</p>
					</div>
					<div class="tabs">
						<ul class="tabs__list">
							<li class="tabs__item tabs__item_active">
								Ремонтный цех
							</li>
							<li class="tabs__item">
								Электрика
							</li>
							<li class="tabs__item">
								Кузовной цех
							</li>
							<li class="tabs__item">
								Автомойка
							</li>
						</ul>
						<div class="tabs__content">
							<div class="tabs__tab">
								<p class="tabs__text">
									Lorem ipsum dolor sit amet,
									cosectetur adipisicing elit,
									sed do eiusmod tempor incididunt cabo
									lorem ipsum dolor sit amet,
									cosectetur adipisicing elit,
									sed do eiusmod tempor incididunt cabo
								</p>
								<div class="tabs__listing">
									<ul class="tabs__ul pull-left">
										<li class="tabs__li">
											<i class="fa fa-check"></i>
											Наименование
										</li>
										<li class="tabs__li">
											<i class="fa fa-check"></i>
											Наименование
										</li>
										<li class="tabs__li">
											<i class="fa fa-check"></i>
											Наименование
										</li>
										<li class="tabs__li">
											<i class="fa fa-check"></i>
											Наименование
										</li>
									</ul>
									<ul class="tabs__ul pull-left">
										<li class="tabs__li">
											<i class="fa fa-check"></i>
											Наименование
										</li>
										<li class="tabs__li">
											<i class="fa fa-check"></i>
											Наименование
										</li>
										<li class="tabs__li">
											<i class="fa fa-check"></i>
											Наименование
										</li>
										<li class="tabs__li">
											<i class="fa fa-check"></i>
											Наименование
										</li>
									</ul>
								</div>
							</div>
						</div>
					</div>
				</div>
				<div class="col-md-4">
					<div class="gallery pull-right hidden-sm">
						<ul class="gallery_imgs">
							<li class="gallery__one">
								<img src="img/img.jpg" alt="" class="gallery_img">
							</li>
						</ul>
						<div class="gallery__nav">
							<span class="gallery_arrow gallery_arrow_prev">
								<i class="fa fa-angle-left"></i>
							</span>
							<span class="gallery_arrow gallery_arrow_next">
								<i class="fa fa-angle-right"></i>
							</span>
						</div>
					</div>
					<div class="clearfix"></div>
					<div class="order-block pull-right hidden-sm">
						<a href="#" class="act order-block__act">Записаться на осмотр</a>
					</div>
				</div>
			</div>
		</div>
	</section>
	<section id="look">
		<div class="container">
			<div class="row">
				<div class="col-md-12">
					<h2>Как выглядит наш центр</h2>
				</div>
			</div>
			<div class="photos">
				<div class="row">
					<div class="col-md-4 col-sm-4 col-xs-12">
						<img class="img-responsive" src="img/SV200020.JPG" alt="Салон">
					</div>
					<div class="col-md-4 col-sm-4 col-xs-12">
						<img class="img-responsive" src="img/SV200020.JPG" alt="Салон">
					</div>
					<div class="col-md-4 col-sm-4 col-xs-12">
						<img class="img-responsive" src="img/SV200020.JPG" alt="Салон">
					</div>
				</div>
			</div>
		</div>
	</section>
	<section id="map">
		<script type="text/javascript" charset="utf-8" 
		async src="https://api-maps.yandex.ru/services/constructor/1.0/js
		/?um=constructor%3A8b6f6037509821f70d286fa701b01949d6d3d44eb47eac52674ba5661ca0220c&amp;
		width=100%25&amp;height=615&amp;lang=ru_UA&amp;scroll=true"></script>
	</section>
	<footer>
		<div class="container">
			<div class="row">
				<div class="col-md-12 text-center">
					<div class="footer__up">
						<i class="fa fa-caret-up"></i>
					</div>
					<div class="footer__credits">
						ООО "Ремонт Хелиос"
					</div>
				</div>
			</div>
		</div>
	</footer>
	<script src="https://use.fontawesome.com/0279959ac4.js"></script>
</body>
</html>