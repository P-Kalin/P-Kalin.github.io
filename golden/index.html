<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Golden</title>
	<link rel="stylesheet" href="css/main.css">
	<link rel="stylesheet" href="css/fonts.css">
	<link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
</head>
<body>
	<header>
		<div class="container">
			<div class="heading clearfix">
				<img class="logo" src="img/logo.png" alt="Golden" ></img>
				<nav>
					<ul class="menu">
						<li>
							<a href="#">Home</a>
						</li>
						<li>
							<a href="#">Services</a>
						</li>	
						<li>
							<a href="#">Portfolio</a>
						</li>	
						<li>
							<a href="#">About</a>
						</li>	
						<li>
							<a href="#">Contacts</a>
						</li>	
					</ul>
				</nav>
			</div>
			<div class="titles">
				<div class="titles_f">
					Welcome to our studio
				</div>
				<h1>
					It's nice to Meet you
				</h1>
			</div>
			<a class="button" href="#">Tell Me More</a>
		</div>
	</header>

	<section id="services">
		<div class="container">
			<div class="title">
				<h2>
					Services
				</h2>
				<p>
					Proin iaculis purus consequat sem cure.
				</p>
			</div>
			<div class="services clearfix">
				<div class="services_item">
					<img src="img/icon1.png" alt="">
					<h3>E-Commerce</h3>
					<p>
						Proin iaculis purus consequat sem cure digni ssim.
						Donec porttitora entum suscipit eanean rhoncus
						posuere odio in tincidunt.
					</p>
				</div>
				<div class="services_item">
					<img src="img/icon2.png" alt="">
					<h3>Responsive Web</h3>
					<p>
						Proin iaculis purus consequat sem cure digni ssim.
						Donec porttitora entum suscipit eanean rhoncus
						posuere odio in tincidunt.
					</p>
				</div>
				<div class="services_item">
					<img src="img/icon3.png" alt="">
					<h3>Web Security</h3>
					<p>
						Proin iaculis purus consequat sem cure digni ssim.
						Donec porttitora entum suscipit eanean rhoncus
						posuere odio in tincidunt.
					</p>
				</div>
			</div>
		</div>
	</section>
	<section id="portfolio">
		<div class="container">
			<div class="title">
				<h2>
					Our portfolio
				</h2>
				<p>
					Proin iaculis purus consequat sem cure.
				</p>
			</div>
			<div class="works clearfix">
				<img src="img/img1.jpg" alt="">
				<img src="img/img2.jpg" alt="">
				<img src="img/img3.jpg" alt="">
			</div>
		</div>
	</section>
	<section id="about">
		<div class="container">
			<div class="title">
				<h2>
					About Us
				</h2>
				<p>
					Proin iaculis purus consequat sem cure. 
				</p>
			</div>
			<div class="information1">
				<span>
					July 2010 <br>
					Our Humble Beginnings <br>
					<p>
						Proin iaculis purus consequat sem cure <br>
						digni ssim. Donec porttitora entum suscipit <br>
						aenean rhoncus posuere odio in tincidunt. Proin <br>
						iaculis purus consequat sem cure digni <br>
						ssim. Donec porttitora entum suscipit. <br>
					</p>
				</span>
				<img src="img/about1.jpg">
			</div>
			<div class="information2">
				<span>
					January 2011<br>
					Facing Startup Battles <br>
					<p>
						Proin iaculis purus consequat sem cure <br>
						digni ssim. Donec porttitora entum suscipit <br>
						aenean rhoncus posuere odio in tincidunt. Proin <br>
						iaculis purus consequat sem cure digni <br>
						ssim. Donec porttitora entum suscipit aenean <br>
						rhoncus posuere odio in tincidunt. <br>
					</p>
				</span>
				<img src="img/about2.jpg">
			</div>
			<div class="information3">
				<span>
					December 2012 <br>
					Enter The Dark Days <br>
					<p>
						Proin iaculis purus consequat sem cure <br>
						digni ssim. Donec porttitora entum suscipit <br>
						aenean rhoncus posuere odio in tincidunt. Proin <br>
						iaculis purus consequat sem cure digni. <br>
					</p>
				</span>
				<img src="img/about3.jpg">
			</div>
			<div class="information4">
				<span>
					February 2014 <br>
					Our Triumph <br>
					<p>
						Proin iaculis purus consequat sem cure <br>
						digni ssim. Donec porttitora entum suscipit <br>
						aenean rhoncus posuere odio in tincidunt. Proin <br>
						iaculis purus consequat sem cure digni <br>
						ssim. Donec porttitora entum suscipit aenean. <br>
					</p>
				</span>
				<img src="img/about4.jpg">
			</div>
		</div>
	</section>
	<footer>
		<div class="container">
			<span>Â© Copyright 2014 FreebiesXpress.com</span>
		</div>
	</footer>
</body>
</html>
